hudi-common/src/main/java/org/apache/hudi/common/config/LockConfiguration.java

```diff
@@ -35,19 +35,23 @@ public class LockConfiguration implements Serializable {
   public static final String LOCK_ACQUIRE_RETRY_MAX_WAIT_TIME_IN_MILLIS_PROP_KEY = LOCK_PREFIX + "max_wait_time_ms_between_retry";
 
   public static final String LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS_PROP_KEY = LOCK_PREFIX + "client.wait_time_ms_between_retry";
+  public static final String DEFAULT_LOCK_ACQUIRE_CLIENT_RETRY_WAIT_TIME_IN_MILLIS = String.valueOf(5000L);
```

Comment:
we probably need to turn these into ConfigProperty? (idk if there was a reason we could nt do this.) @yihua may know.

Reply:
Wrt this PR, is this refactoring existing code?  We have common lock configs in `HoodieLockConfig` class.

Reply:
@danny0405 @boneanxs ^

Reply:
`TimeGeneratorBase` need this default value, it cannot access `HoodieLockConfig` since it's not in `hudi-common`